spec/fixtures/integration/user_defined_parameterizing_rules.y

```diff
@@ -21,10 +21,11 @@ static int yyerror(YYLTYPE *loc, const char *str);
 
 %type <num> stmt
 
-%rule pair(X, Y): X Y
+%rule pair(X, Y): X Y[alias]
```

Comment:
Is `[alias]` needed?

Reply:
Indeed it is. Thank you! I updated this PR.